block/internal/syncing/p2p_handler.go

```diff
@@ -0,0 +1,208 @@
+package syncing
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	goheader "github.com/celestiaorg/go-header"
+	"github.com/rs/zerolog"
+
+	"github.com/evstack/ev-node/block/internal/cache"
+	"github.com/evstack/ev-node/block/internal/common"
+	"github.com/evstack/ev-node/pkg/genesis"
+	"github.com/evstack/ev-node/types"
+)
+
+// P2PHandler handles all P2P operations for the syncer
+type P2PHandler struct {
+	headerStore goheader.Store[*types.SignedHeader]
+	dataStore   goheader.Store[*types.Data]
+	cache       cache.Manager
```

Comment:
This attribute is unused and can be deleted

Reply:
https://github.com/evstack/ev-node/pull/2644/commits/90a028f08df9e643192396ffeb15e9ef35d7b5a0